### PR TITLE
fix: static analysis errors

### PR DESCRIPTION
**Pull request by millnut**
## What does this change?

Fixes (#87) errors reported by the static analysis check

### CODE REVIEW

File: src/AddressLookup.php
Comment (Adnan-cds) [inline src/AddressLookup.php:1]:
```
diff --git a/src/AddressLookup.php b/src/AddressLookup.php
index 5aefc2e..9941a5c 100644
--- a/src/AddressLookup.php
+++ b/src/AddressLookup.php
@@ -102,18 +102,18 @@ class AddressLookup {
     }

     $address = [
-      'name'         => $unique_id ?? '',
-      'uprn'         => $uprn ?? '',
-      'display'      => $display_name ?? '',
+      'name'         => $unique_id,
+      'uprn'         => $uprn,
+      'display'      => $display_name,
       'street'       => implode(' ', array_filter([
         $street_number, $street_name,
       ])),
-      'flat'         => $flat ?? '',
+      'flat'         => $flat,
       'house'        => implode(', ', array_filter([$org, $house_name])),
       'town'         => $locality ?? '',
       'postcode'     => $postcode ?? '',
-      'lat'          => $latitude ?? '',
-      'lng'          => $longitude ?? '',
+      'lat'          => $latitude ?: '',
+      'lng'          => $longitude ?: '',
       'country'      => $country_name ?? '',
       'country_code' => $country_code ?? '',
       'line1'        => '',
```

**Review by Adnan-cds — COMMENTED** [review]
Hi Lee,
Thanks for looking into this.  I have finally managed to configure phpstan to highlight these errors :)  Your fixes look okay.  I have proposed a slightly different fix in one of the files after examining the relevant code.  Let me know what you think.

File: src/AddressLookup.php
Comment (millnut) [reply src/AddressLookup.php:1]: Thanks @Adnan-cds I've added those changes to the PR

**Review by Adnan-cds — APPROVED** [review]
Thank you Lee.  All good now.

**Pull request merged**